data/default_release_images.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "openshift_version": "4.6",
+        "cpu_architecture": "x86_64",
```

Comment:
should we add some `arm` example? Or would it fail if we add it now?

Reply:
Yeah, multiple architectures not yet supported by this PR. So I'll add an arm image in an upcoming PR when adding the 4.9 arm64 release image.